dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -19,18 +19,28 @@
 
 package org.apache.cxf.message;
 
+import java.util.AbstractMap;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
+import javax.servlet.http.HttpServletResponse;
+
 import org.apache.cxf.Bus;
+import org.apache.cxf.configuration.security.AuthorizationPolicy;
+import org.apache.cxf.continuations.ContinuationProvider;
 import org.apache.cxf.endpoint.Endpoint;
-import org.apache.cxf.helpers.CastUtils;
 import org.apache.cxf.interceptor.InterceptorChain;
+import org.apache.cxf.interceptor.OutgoingChainInterceptor;
+import org.apache.cxf.phase.PhaseInterceptorChain;
+import org.apache.cxf.security.SecurityContext;
 import org.apache.cxf.service.Service;
 import org.apache.cxf.service.model.EndpointInfo;
 import org.apache.cxf.transport.Destination;
+import org.apache.cxf.transport.http.AbstractHTTPDestination;
+import org.apache.cxf.transport.https.CertConstraints;
 
 public class MessageImpl extends StringMapImpl implements Message {
```

Comment:
I think there are additional methods that need to be implemented:  size, clear, isEmpty, containsValue, equals, hashCode, getOrDefault, forEach, replaceAll, putIfAbsent, remove(Object, Object), replace, computeIfAbsent, computerIfPresent, compute, merge.  We may want to just implement them and have them return UnsupportedOperationException if we don't think they are called by CXF.

Reply:
So you implemented some as throwing UnsupportedOperationException. Was that intentional or you plan to implement those still?